<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptContentDialog</class>
 <widget class="QDialog" name="ScriptContentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">[replace]</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="scriptContent">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clipboardButton">
       <property name="text">
        <string notr="true">[clipboard action here]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Select all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScriptContentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScriptContentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>scriptContent</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
